epub33/core/vocab/item-properties.html

```diff
@@ -1,18 +1,15 @@
 <section id="app-item-properties-vocab" class="vocab">
 	<h3>Manifest Properties Vocabulary</h3>
-	
 	<p>The properties in this vocabulary are usable in the manifest <a href="#sec-item-elem"><code>item</code>
 			element's</a>
 		<a href="#attrdef-item-properties"><code>properties</code> attribute</a>.</p>
-	
 	<p>The <strong>Applies to</strong> field indicates which Publication Resource type(s) the given property MAY
 		be specified on, the <strong>Cardinality</strong> field indicates the number of times the property MUST
 		appear within the Package Document scope, and the <strong>Usage</strong> field indicates usage
 		conditions.</p>
-	
-	<p>Properties without a prefix are referenceable using the base IRI
+	<p>Properties without a prefix are referenceable using the <a
+			href="https://url.spec.whatwg.org/#concept-base-url">base URL</a> [[URL]]
 			<code>http://idpf.org/epub/vocab/package/item/#</code>.</p>
```

Comment:
I'm going to change these to match the rendition vocab's: "The base URL for referencing these properties is ...".

I'm also going to link them up to the default vocabulary section and modify this sentence as follows so it's clearer why we have these base URLs defined:

> EPUB Creators MUST NOT assign a prefix to the ___base___ URLs ___[[URL]]___ associated with these vocabularies using the prefix attribute.

Otherwise, I expect we'll just confuse people more why we have URLs for properties that don't need or have prefixes.